src/lexers/jsx-lexer.js

```diff
@@ -79,7 +85,7 @@ export default class JsxLexer extends JavascriptLexer {
           }
         }
       },
-      JSXParserExtension
+      this.WalkerBase
```

Comment:
The changes here are to utilize the inheritable properties from the JavascriptLexer. 

- extend WalkerBase with the JSX Extensions
- add the JSX plugin config value to acornConfig
- inject the JSX plugin to acorn itself